examples/test/GeoPushpinChart_test.js

```diff
@@ -0,0 +1,57 @@
+// (C) 2020 GoodData Corporation
+import { Selector } from "testcafe";
+import { config } from "./utils/config";
+import { loginUsingLoginForm } from "./utils/helpers";
+import { DEFAULT_COLOR_PALETTE } from "../src/utils/colors";
+
+fixture("Geo Pushpin Chart Demo")
+    .page(config.url)
+    .beforeEach(loginUsingLoginForm(`${config.url}/geo-pushpin-chart`));
+
+const CHART_WITH_CLUSTERING = Selector(".s-geo-pushpin-chart-clustering");
+const CHART_WITH_COLOR_LEGEND = Selector(".s-geo-pushpin-chart-color");
+const CHART_WITH_CETEGORY_LEGEND = Selector(".s-geo-pushpin-chart-category");
+const CHART_WITH_CONFIGURATION = Selector(".s-geo-pushpin-chart-configuration");
+
+test("should render all charts", async t => {
+    await t.expect(CHART_WITH_CLUSTERING.exists).ok();
+    await t.expect(CHART_WITH_COLOR_LEGEND.exists).ok();
+    await t.expect(CHART_WITH_CETEGORY_LEGEND.exists).ok();
+    await t.expect(CHART_WITH_CONFIGURATION.exists).ok();
+
+    await t.expect(CHART_WITH_CLUSTERING.find("canvas").exists).ok();
+    await t.expect(CHART_WITH_COLOR_LEGEND.find("canvas").exists).ok();
+    await t.expect(CHART_WITH_CETEGORY_LEGEND.find("canvas").exists).ok();
+    await t.expect(CHART_WITH_CONFIGURATION.find("canvas").exists).ok();
+});
+
+test("should not render the legend with Clustering", async t => {
+    await t.expect(CHART_WITH_CLUSTERING.find(".geo-legend").exists).notOk();
```

Comment:
do we have the `s-` prefix for these classnames?

Reply:
fixed

Reply:
@dlcd2203 please apply the same for `color-legend` and `pushpin-size-legend `

Reply:
we have 's-geo-legend ' class is a parent element for color, size, category legend.